python/lsst/sims/maf/stackers/NEODistStacker.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+from .baseStacker import BaseStacker
+
+
+__all__ = ['NEODistStacker']
+
+class NEODistStacker(BaseStacker):
+    """
+    For each observation, find the max distance to a ~144 km NEO,
+    also stack on the x,y position of the object.
+    """
+
+    def __init__(self, m5Col='fiveSigmaDepth',
+                 stepsize=.001, maxDist=3.,minDist=.3, H=22, elongCol='solarElong',
+                 filterCol='filter',sunAzCol='sunAz', azCol='azimuth', **kwargs):
+
+        """
+        stepsize:  The stepsize to use when solving (in AU)
+        maxDist: How far out to try and measure (in AU)
+        H: Asteroid magnitude
+
+        Adds columns:
+        NEOGeoDist:  Geocentric distance to the NEO
```

Comment:
Is this the maxGeoDist that we could see an NEO to? Would you have strong feelings about renaming it NEOMaxGeoDist instead of just GeoDist? 
MaxGeoDist just seems clearer than GeoDist, particularly if you're thinking about a population of NEOs (which would all have their independent GeoDist's).